sonaranalyzer-dotnet/its/expected/akka.net/Akka.Remote.TestKit-{E5957C3E-2B1E-469F-A680-7953B4DEA31B}-S4487.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
+{
+"id":  "S4487",
+"message":  "Remove this unread private field '_log' or refactor the code to use its value.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka.Remote.TestKit\MultiNodeSpec.cs",
+"region":  {
+"startLine":  380,
+"startColumn":  34,
+"endLine":  380,
+"endColumn":  38
+}
+}
+},
+{
+"id":  "S4487",
+"message":  "Remove this unread private field '_handler' or refactor the code to use its value.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka.Remote.TestKit\Player.cs",
```

Comment:
@alban-auzeill  - this is 
```
        readonly PlayerHandler _handler;
```
thus internal

Reply:
from @alban-auzeill  - there has to be a bug with the map...

Reply:
this is still a problem, as it's internal, but given we use the Roslyn API to make sure the accessibility is private, there's not much more we can do